Formula/c/cargo-edit.rb

```diff
@@ -3,8 +3,8 @@ class CargoEdit < Formula
   homepage "https://killercup.github.io/cargo-edit/"
   # TODO: check if we can use unversioned `libgit2` at version bump.
   # See comments below for details.
```

Comment:
I would assume the CI failure is probably related to this, but unfortunately I don't know enough about brew to really help :sweat_smile:

Reply:
The comments below describe exactly what you needed to do in detail. I've done that for you now.

Reply:
Was slowly working through that myself, but glad that you handled it for me. Thanks :heart:

Reply:
Huh, looks like it'll take bit more than what was in the comments. I'll have a look again tomorrow.